additional_recipes/ros-humble-dynmsg/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+package:
+  name: ros-humble-dynmsg
+  version: "1.0.0"
+
+source:
+  git: https://github.com/osrf/dynamic_message_introspection.git
+  rev: 50277e341f61455c42a6015b5718804a4f8588f6
+  target_directory: ros-humble-dynmsg/src/work
+  patches:
+    - fix-build.patch
+
+build:
+  number: 12
+  script:
+    build.sh
+
+requirements:
+  build:
+    - if: target_platform == 'emscripten-wasm32'
+      then:
+        - emscripten_emscripten-wasm32
+      else: 
+        - ${{ compiler('cxx') }}
+        - ${{ compiler('c') }}
+    - if: linux64
+      then:
+        - sysroot_linux-64 2.17
+    - if: unix
+      then:
+        - make
+        - coreutils
+    - if: osx
+      then: tapi
+    - if: build_platform != target_platform
+      then:
+        - pkg-config
+        - python
+        - cross-python_${{ target_platform }}
+        - cython
+        - numpy
+    - ninja
+    - setuptools
+    - cmake
+  host:
+    - numpy
+    - pip
+    - if: build_platform == target_platform
+      then:
+        - pkg-config
+    - python
+    - ros-humble-ament-cmake
+    - ros-humble-rcutils
+    - ros-humble-rosidl-runtime-c
+    - ros-humble-rosidl-runtime-cpp
+    - ros-humble-rosidl-typesupport-introspection-c
+    - ros-humble-rosidl-typesupport-introspection-cpp
+    - ros-humble-yaml-cpp-vendor
+    - ros2-distro-mutex 0.6.* humble_*
+  run:
+    - if: target_platform != 'emscripten-wasm32'
+      then:
+        - cmake
+    - python
+    - ros-humble-rosidl-runtime-c
+    - ros-humble-rosidl-runtime-cpp
+    - ros-humble-rosidl-typesupport-introspection-c
+    - ros-humble-rosidl-typesupport-introspection-cpp
+    - ros-humble-yaml-cpp-vendor
+    - ros2-distro-mutex 0.6.* humble_*
```

Comment:
I am still afraid of this, but I tried to mitigate this in https://github.com/RoboStack/robostack.github.io/pull/82 .